src/js/components/alert.js

```diff
@@ -5,43 +5,67 @@
 
 export default class Alert {
 
-  constructor(options) {
-    this.handleClick = this.handleClick.bind(this);
+  constructor(element) {
 
-    this.init(options);
-  }
-
-  init(options) {
-    document.getElementById(options.id).closest('[aria-labelledby]').querySelector('[data-alert-close]').addEventListener('click', () => { this.handleClick(options.id); }, false);
-  }
+    // Instance properties
+    this.element = element;
+    this.closeAttribute = ['data-alert-close'];
```

Comment:
This is setting the `closeAttribute` class property to an array with one item (a string of `data-alert-close`) in it. 

It should be `this.closeAttribute = 'data-alert-close';` and then you can set `this.closeSelector = `[${this.closeAttribute}]` to the correct attribute CSS selector.

The reason you want the `this.closeAttribute` property to be only a string that is the data attribute name is that you may need to use it on it's own for some DOM methods e.g. `element.getAttribute(this.closeAttribute)`. In this case you con't need the fully-formed CSS selector string (with brackets `[]`), just the name of the attribute.

If you keep the attribute stored separately you can then interpolate it into the CSS selector you need later. e.g. `this.closeSelector = `[${this.closeAttribute}]` which gives you `[data-alert-close]` which you could use with `document.querySelector()`.